src/include/common/spin_latch.h

```diff
@@ -0,0 +1,57 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Remove this designation as we haven't decided on what to do with them in the new codebase yet.